pages/blog/2024-march-summary.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Community Update: March 2024"
+date: 2024-04-04T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/2024-blog-banner/blog-banner-march.webp
+authors:
+  - name: Thulisile Sibanda
+    photo: /img/avatars/thulieblack.webp
+    link: https://www.linkedin.com/in/v-thulisile-sibanda/
+    byline: AsyncAPI Community Manager
+excerpt: 'Community update for March'
+featured: true
+---
+We are excited to announce and welcome our new Silver sponsor, [Aklivity](https://www.aklivity.io/), to the AsyncAPI Initiative.
+
+
+<a href='https://www.aklivity.io/' target='_blank'>
+<img src='/img/sponsors/aklivity_logo.png' alt='aklivity' width='500px' />
+</a>
+
+Additionally, we welcome back [RedHat](https://www.redhat.com/en) back as our Bronze sponsor for 2024. We extend our gratitude to [Hugo Guerrero](https://www.linkedin.com/in/hugo-guerrero), our AsyncAPI Ambassador, for connecting the right people from RedHat.
+
+<a href='https://www.redhat.com/' target='_blank'>
+<img src='/img/sponsors/redhat.svg' alt='redhat' width='500px' />
+</a>
+
+We encourage organizations to support the initiative by getting involved either financially, through employee participation, service provision, or hosting an AsyncAPI event. Please refer [to our AsyncAPI Financial Summary](https://www.asyncapi.com/finance) for a detailed breakdown.
+
+
+## AsyncAPI Conf on Tour
+The call for speakers for [AACoT'24 Helsinki Edition](https://conference.asyncapi.com/venue/Helsinki) has ended, and we will soon add the speakers, agenda, and ticket information [to the AsyncAPI conference website](conference.asyncapi.com). 
+
+If you missed the deadline for submitting a talk, we are happy to share that the proposals for AACoT'24 London will soon be open. Be sure to start drafting your proposals and prepare to join us in September.
+
+## Spec X Tooling
+It's been a while since we released the [AsyncAPI v3](https://www.asyncapi.com/blog/release-notes-3.0.0), and its adoption and support over the past few months have been amazing. We are happy to share some of the projects that are doing cool stuff with v3:
```

Comment:
```suggestion
It's been a while since we released [AsyncAPI v3](https://www.asyncapi.com/blog/release-notes-3.0.0), and its adoption and support over the past few months have been amazing. We are happy to share some of the projects that are doing cool stuff with v3:
```